src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
-body {
-  margin: 0;
+@import "./utils/_variables";
+@import "./utils/normalize";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+  width: calc(7 * #{$cell-size } + 18px);
```

Comment:
dont write hardcoded value like 18 px
here with `$cell-size ` consider gap between cells and calender paddings on both side

Reply:
done.